packages/move/sources/vesting.move

```diff
@@ -124,6 +158,11 @@ module deployment_addr::vesting {
         vesting_stream.claimed_amount = vesting_stream.claimed_amount + claimable_tokens;
     }
 
+    // - - - CREATE STREAM - - -
+
+    // FIX: given the above recommended fixes the create_vesting_stream function would have to be adjusted to allow the creation of arbitrary Object<VestingStream> instances
+    // The current implementation only allows a given benefiary to have only a single stream.
```

Comment:
First of all, the check for existing stream should be removed. 
Second, to allow multiple streams per beneficiary, streams should not be keyed to an address in global storage.